idea/idea-core/src/org/jetbrains/kotlin/idea/core/psiModificationUtils.kt

```diff
@@ -70,7 +72,11 @@ fun KtLambdaArgument.moveInsideParenthesesAndReplaceWith(
     val newCallExpression = oldCallExpression.copy() as KtCallExpression
 
     val psiFactory = KtPsiFactory(project)
-    val argument = if (newCallExpression.getValueArgumentsInParentheses().any { it.isNamed() }) {
+
+    val argument = if (newCallExpression.getValueArgumentsInParentheses().run {
```

Comment:
I agree @yole. This part is really good example what should be extracted to method.